config/containers/logging/dual-logging.md

```diff
@@ -0,0 +1,116 @@
+---
+description: Learn how to read container logs locally when using a third party logging solution.
+keywords: docker, logging, driver
+title: Reading container logs locally with remote logging drivers
+---
+
+## Overview 
+
+In previous versions of Docker Enterprise, the jsonfile and journald log drivers supported reading 
+container logs locally using `docker logs`. However, many third party logging drivers had no 
+support for locally reading logs using `docker logs`, including: 
+
+- syslog	
+- gelf	
+- fluentd	
+- awslogs	
+- splunk	
+- etwlogs	
+- gcplogs	
+- Logentries
+
+This created multiple problems, especially with UCP, when attempting to gather log data in an 
+automated and standard way. Log information could only be accessed and viewed through the 
+third-party solution in the format specified by that third-party tool, which often required authentication. 
+
+Starting with Docker Engine Enterprise 18.03.1-ee-1, you can use `docker logs` to read container 
+logs regardless of the configured logging solution or plugin. This capability, sometimes referred to 
```

Comment:
"configured logging solution or plugin" should be
"configured logging driver or plugin"

Reply:
Got it.